configuration/arm64/initialization.sh

```diff
@@ -9,9 +9,6 @@ sudo apt update -y && sudo apt upgrade -y
 sudo apt-get install chromium-browser -y      #Chrome required for CARMA platform/V2X Hub UI(?)
 sudo apt install curl -y                      #Curl for downloading files over internet
 
-#install docker
-curl -L https://raw.githubusercontent.com/usdot-fhwa-stol/carma-platform/develop/engineering_tools/install-docker.sh | bash 
```

Comment:
Why is this removed

Reply:
Why was this ever here.

Reply:
Can this be removed everywhere

Reply:
@paulbourelly999 I'm not sure why it was added, but Docker is installed below using the same steps in this script. I can remove it from the amd64 file, too.